logcat/src/androidTest/kotlin/AndroidLogcatTest.kt

```diff
@@ -0,0 +1,40 @@
+package logcat
+
+import logcat.LogPriority.INFO
+import kotlin.test.AfterTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class AndroidLogcatTest {
+
+  @AfterTest
+  fun tearDown() {
+    LogcatLogger.uninstall()
+  }
+
+  @Test fun `logcat() passes priority to isLoggable check`() {
```

Comment:
Parens in a function name seems... Needlessly dangerous

Reply:
Changing to `logcat fun passes priority to isLoggable check`